automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/runtimes/RuntimeGceSpec.scala

```diff
@@ -76,58 +82,56 @@ class RuntimeGceSpec extends BillingProjectFixtureSpec with ParallelTestExecutio
     res.unsafeRunSync()(cats.effect.unsafe.IORuntime.global)
   }
 
-  // Not enable this in automation test because we can get `ZONE_RESOURCE_POOL_EXHAUSTED` easily
-  // IA-4724
-//  "should be able to create a VM with GPU enabled" ignore { project =>
-//    val runtimeName = randomClusterName
-//    val diskName = genDiskName.sample.get
-//
-//    val toolImage = ContainerImage.fromImageUrl(
-//      "us.gcr.io/broad-dsp-gcr-public/terra-jupyter-python:qi-gpu"
-//    ) // Use the default base image once we migrate to use gpu images by default
-//    // In a europe zone
-//    val createRuntimeRequest = defaultCreateRuntime2Request.copy(
-//      runtimeConfig = Some(
-//        RuntimeConfigRequest.GceWithPdConfig(
-//          Some(MachineTypeName("n1-standard-4")),
-//          PersistentDiskRequest(
-//            diskName,
-//            None,
-//            None,
-//            Map.empty
-//          ),
-//          Some(ZoneName("us-west1-a")),
-//          Some(GpuConfig(GpuType.NvidiaTeslaT4, 2))
-//        )
-//      ),
-//      toolDockerImage = toolImage
-//    )
-//// TODO: investigate, do we need to use a notebook to validate this? can we use a command?
-//    val res = dependencies.use { deps =>
-//      implicit val httpClient = deps.httpClient
-//      for {
-//        runtime <- LeonardoApiClient.createRuntimeWithWait(project, runtimeName, createRuntimeRequest)
-//        clusterCopy = ClusterCopy.fromGetRuntimeResponseCopy(runtime)
-//        implicit0(authToken: AuthToken) <- Ron.authToken()
-//        _ <- IO(withWebDriver { implicit driver =>
-//          withNewNotebook(clusterCopy, Python3) { notebookPage =>
-//            val deviceNameOutput =
-//              """
-//                |import tensorflow as tf
-//                |gpus = tf.config.experimental.list_physical_devices('GPU')
-//                |print(gpus)
-//                |""".stripMargin
-//            val output = notebookPage.executeCell(deviceNameOutput).get
-//            output.contains("GPU:0") shouldBe true
-//            output.contains("GPU:1") shouldBe true
-//          }
-//        })
-//        _ <- LeonardoApiClient.deleteRuntime(project, runtimeName)
-//      } yield ()
-//    }
-//
-//    res.unsafeRunSync()(cats.effect.unsafe.IORuntime.global)
-//  }
+  "should be able to create a VM with GPU enabled" taggedAs ScheduledTest in { project =>
+    val runtimeName = randomClusterName
+    val diskName = genDiskName.sample.get
+
+    val toolImage = ContainerImage.fromImageUrl(
+      "us.gcr.io/broad-dsp-gcr-public/terra-jupyter-python:qi-gpu"
+    ) // Use the default base image once we migrate to use gpu images by default
+    // In a europe zone
```

Comment:
is this part of the TODO? isn't the zone us-west1-a?

Reply:
Ohhhh, very good catch, thanks! Damn that test is old 😰